src/stores/PathDetailsStore.ts

```diff
@@ -0,0 +1,56 @@
+import Store from "@/stores/Store";
+import {Action} from "@/stores/Dispatcher";
+import {PathDetailsElevationSelected, PathDetailsHover, PathDetailsRangeSelected} from "@/actions/Actions";
+import {Coordinate} from "@/stores/QueryStore";
+import {Bbox} from "@/api/graphhopper";
+
+export interface PathDetailsPoint {
+    point: Coordinate
+    elevation: string
+    description: string
+}
+
+export interface PathDetailsStoreState {
+    pathDetailsPoint: PathDetailsPoint | null
+    pathDetailBbox?: Bbox
+    pathDetailsHighlightedSegments: Coordinate[][]
+}
+
+export default class PathDetailsStore extends Store<PathDetailsStoreState> {
+    constructor() {
+        super()
+    }
+
+    protected getInitialState(): PathDetailsStoreState {
+        return {
+            pathDetailsPoint: null,
+            pathDetailBbox: undefined,
+            pathDetailsHighlightedSegments: []
+        }
+    }
+
+
+    reduce(state: PathDetailsStoreState, action: Action): PathDetailsStoreState {
+        if (action instanceof PathDetailsHover) {
+            return {
+                ...state,
+                pathDetailsPoint: action.pathDetailsPoint
+            }
+        } else if (action instanceof PathDetailsRangeSelected) {
+            return {
+                // todo: bug - when we change details and selected a range before and then zoomed somewhere else we jump
+                //       back to the box unintentionally
+                ...state,
```

Comment:
This is still WIP..., but I opened the PR so maybe we can already fix some of the layout stuff

Reply:
This was a problem in heightgraph and I fixed it there